stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -59,14 +59,16 @@ public type Client client object {
     #
     # + sqlQuery - SQL statement to execute
     # + parameters - The parameters to be passed to the update query
+    # + getGeneratedKeys - Allow to retrieves auto-generated keys after a statement has been executed if value is
+    #                      `true`.It will return auto-generated keys only if supported by JDBC
     # + return - `UpdateResult` with the updated row count and key column values,
     #             else `Error` will be returned if there is an error
-    public remote function update(@untainted string sqlQuery, Param... parameters)
-                                  returns UpdateResult|Error {
+    public remote function update(@untainted string sqlQuery, boolean? getGeneratedKeys = (),
+                                  Param... parameters) returns UpdateResult|Error {
         if (!self.clientActive) {
             return self.handleStoppedClientInvocation();
         }
-        return nativeUpdate(self, java:fromString(sqlQuery), parameters);
+        return nativeUpdate(self, java:fromString(sqlQuery), parameters, getGeneratedKeys);
```

Comment:
Shall we swap `parameters` and `getGeneratedKeys`? Bascially have the same order in the remote function it self.

Reply:
We  can't swap as required parameter not allowed after default-able parameters.
ERROR: ballerinax/java.jdbc:0.0.0::client.bal:132:24:: required parameter not allowed after defaultable parameters

Reply:
We don't need to make the parameter defaultable in internal function. because we always have a value for that parameter.